uxarray/grid/grid.py

```diff
@@ -1221,3 +1234,139 @@ def isel(self, **dim_kwargs):
             raise ValueError(
                 "Indexing must be along a grid dimension: ('n_node', 'n_edge', 'n_face')"
             )
+
+    def _populate_bounds(self, is_latlonface: bool = False, is_face_GCA_list=None):
```

Comment:
To answer your question about where to include this in the documentation, this shouldn't be under our `Grid`

We should move it into either `geometry.py` or `arcs.py` depending on what you see more fit.

See how we do this in `connectivity.py` as a reference.

Reply:
The `Grid._populate_bounds()` API was already there under our `Grid` when I had the API draft

But I can proceed to change that into the `geometry.py`

Reply:
I see! The draft API was written a while before we made made that design choice, so that makes sense.

`geometry.py` will be a good fit for it then.